src/main/java/uk/gov/hmcts/dts/fact/model/Contact.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.dts.fact.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@JsonPropertyOrder({"number", "description", "explanation"})
+public class Contact {
+    private String number;
+    @JsonProperty("description")
+    private String name;
+    private String explanation;
+
+    public Contact(uk.gov.hmcts.dts.fact.entity.Contact contact) {
```

Comment:
could this not be imported and become... (Contact contact) {

Reply:
need to differentiate between uk.gov.hmcts.dts.fact.entity.* and uk.gov.hmcts.dts.fact.model.*